libcontainer/cgroups/fs/memory.go

```diff
@@ -108,28 +121,28 @@ func setMemoryAndSwap(path string, cgroup *configs.Cgroup) error {
 		// for memory and swap memory, so it won't fail because the new
 		// value and the old value don't fit kernel's validation.
 		if memoryUsage.Limit < uint64(cgroup.Resources.MemorySwap) {
```

Comment:
If we set memsw to -1 without changing memlimit I don't think it fails, since memory will be '0' (means no change) and it just sets memsw to -1 (which means unlimited)

Did I miss something here?